go.mod

```diff
@@ -220,6 +220,8 @@ require (
 
 replace github.com/ethereum/go-ethereum v1.13.8 => github.com/ethereum-optimism/op-geth v1.101308.2-rc.2
 
+replace github.com/ethereum-optimism/optimism => ./
```

Comment:
I think you don't have to add this line! 🙇🏼‍♂️
If I am wrong, let me know! 🙇🏼‍♂️

Reply:
Thank you very much, Steven
You are right!